server/paymentProviders/transferwise/index.ts

```diff
@@ -243,24 +248,25 @@ async function createExpensesBatchGroup(
   });
 
   try {
-    const transferIds = await Promise.all(
-      expenses.map(async expense => {
-        const { transfer } = await createTransfer(connectedAccount, expense.PayoutMethod, expense, {
+    const transfers = await Promise.all(
+      expenses.map(expense =>
+        createTransfer(connectedAccount, expense.PayoutMethod, expense, {
           batchGroupId: batchGroup.id,
           token,
-        });
-        return transfer.id;
-      }),
+        })
+          .then(({ transfer }) => transfer.id)
+          .catch(() => {
+            // createTransfer will automatically mark the Expense as error and comment the issue
+            // Here we just ignore so we can pay the rest of the batch.
+          }),
```

Comment:
@kewitz : Just a suggestion, but my thinking here is to just log the exception (without throwing it) as having this empty block might cause some hard to find bugs down the pipe. For example; if an error happens at `Expense.prototype.createActivity` without this logging it will not show the real error anywhere but rather just the, `All expenses failed, there is no batch to pay.`

Reply:
This catch belongs to a single expense promise chain. Notice that createTransfer is already taking care of logging its errors.

Reply:
@kewitz : Yes, `createTransfer` is already taking care of most of the logging, except within the `createTransfer` catch block; 

https://github.com/opencollective/opencollective-api/blob/5a577324a84a056f7e2a7bc398219e1773603d4a/server/paymentProviders/transferwise/index.ts#L184-L189

we have `createActivity`. If that throws the error it won't be printed anywhere I think. Hope that makes sense. Let me know if I am missing something here. 😄